src/services/apiService.js

```diff
@@ -171,13 +171,54 @@ export const getPieceById = async (id) => {
   }
 };
 
+export const fetchCollections = async () => {
+  try {
+    // Request a single page of 100 painting items from the Rijksmuseum API
+    const response = await axios.get(
+      `https://www.rijksmuseum.nl/api/en/collection`,
+      {
+        params: {
+          key: "A3B8HWv5",

```

Comment:
That is not a good practise to hardcode Api keys. Kindly change that

Reply:
Corrected this part of code in my latest commit, thanks for pointing it out!